glean-core/ffi/examples/glean_app.c

```diff
@@ -45,14 +45,15 @@ int main(void)
   // NOTE: If, there are other ping files inside tmp/glean_data directory
   // they will also be consumed here by `glean_process_ping_upload_response`.
   FfiPingUploadTask task;
-  glean_get_upload_task(&task, 1);
+  glean_set_log_pings(1);
+  glean_get_upload_task(&task);
 
   while (task.tag != FfiPingUploadTask_Done) {
       printf("tag: %d\n", task.tag);
 
       if (task.tag == FfiPingUploadTask_Upload) {
           printf("path: %s\n", task.upload.path);
-          printf("body length: %lld\n", task.upload.body.len);
+          printf("body length: %d\n", task.upload.body.len);
```

Comment:
I was getting a warning about this everytime I built the C app. Was there a readon for the previous choice?

Reply:
Which warning?

Reply:
```
cc  -I.. -o glean_app glean_app.c ../../../target/debug/libglean_ffi.dylib
glean_app.c:55:41: warning: format specifies type 'long long' but the argument has type
      'int32_t' (aka 'int') [-Wformat]
          printf("body length: %lld\n", task.upload.body.len);
                               ~~~~     ^~~~~~~~~~~~~~~~~~~~
                               %d
```